src/App.tsx

```diff
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
It's better to move any "magic numbers" to the separate constant. 
For example:
```
const INITIAL_PER_PAGE = 5;
const INITIAL_PAGE = 1;
```